### PR TITLE
More miscellaneous fixes for non-long double support

### PR DESCRIPTION
**Pull request by embray**
Should be fairly self-explanatory; thanks in advance?

### CODE REVIEW

File: src/fpylll/fplll/pruner.pyx [hunk 1]
```diff
@@ -35,6 +35,10 @@ from fplll cimport svp_probability as svp_probability_c
 from fpylll.util import adjust_radius_to_gh_bound, precision, get_precision
 from fpylll.util cimport check_float_type, check_precision, check_descent_method, check_pruner_metric
 
+IF HAVE_LONG_DOUBLE:
+    from fplll import FT_LONG_DOUBLE
+    from decl import mpz_ld
```
Comment (embray) [inline src/fpylll/fplll/pruner.pyx]: I noticed that this import isn't used explicitly, nor or some of the similar ones on line 25, but I kept it anyways in case there was something non-obvious I was missing.

File: src/fpylll/fplll/decl.pxd [hunk 2]
```diff
@@ -228,12 +228,12 @@ ELSE:
...
             FastEvaluator[FP_NR[dd_real]] *dd
             FastEvaluator[FP_NR[qd_real]] *qd
-            FastEvaluator[FP_NR[mpfr_t]] *mpfr
+            FastErrorBoundedEvaluator *mpfr
     ELSE:
         ctypedef union fast_evaluator_core_t:
             FastEvaluator[FP_NR[double]] *double
             FastEvaluator[FP_NR[dpe_t]] *dpe
-            FastEvaluator[FP_NR[mpfr_t]] *mpfr
+            FastErrorBoundedEvaluator *mpfr
```
Comment (malb) [inline src/fpylll/fplll/decl.pxd]: Why this change?

Comment (embray) [inline src/fpylll/fplll/decl.pxd]: I don't know, but it's the same change that was made in 3ca12c7528b5cfebd34d2421b119133b4f03942c, but just wasn't propagated to this branch if the IF/ELSE

Comment (embray) [inline src/fpylll/fplll/decl.pxd]: (The Cython module doesn't compile without it)

Comment (malb) [inline src/fpylll/fplll/decl.pxd]: Ah, of course, ta!

File: src/fpylll/fplll/pruner.pyx (hunk 1 above)
Comment (malb) [inline src/fpylll/fplll/pruner.pyx]: this should read `cimport`

Comment (embray) [inline src/fpylll/fplll/pruner.pyx]: Oops--thanks.
